packages/cli/src/operations.ts

```diff
@@ -1,16 +1,23 @@
 import { transformOas3Operations } from '@stoplight/http-spec/oas3/operation';
 import { transformOas2Operations } from '@stoplight/http-spec/oas2/operation';
 import { transformPostmanCollectionOperations } from '@stoplight/http-spec/postman/operation';
-import { dereference } from '@stoplight/json-schema-ref-parser';
+import { dereference, HTTPResolverOptions } from '@stoplight/json-schema-ref-parser';
 import { bundleTarget, decycle } from '@stoplight/json';
 import { IHttpOperation } from '@stoplight/types';
 import { get } from 'lodash';
+import * as os from 'os';
 import type { Spec } from 'swagger-schema-official';
 import type { OpenAPIObject } from 'openapi3-ts';
 import type { CollectionDefinition } from 'postman-collection';
 
 export async function getHttpOperationsFromSpec(specFilePathOrObject: string | object): Promise<IHttpOperation[]> {
-  const result = decycle(await dereference(specFilePathOrObject));
+  const prismVersion = require('../package.json').version;
+  const httpResolverOpts: HTTPResolverOptions = {
+    headers: {
+      'User-Agent': `Prism Mock Server v${prismVersion} (${os.type()} ${os.arch()} ${os.release()})`,
```

Comment:
i'd make sure we follow the semistandard format for this: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent

Reply:
Can you please be more specific with the requirements since a format wasn't originally specified in the ticket? Are you wanting 'prism/<version>`?>

Reply:
I think that's what @daniel-white meant, like `curl/7.64.1` or `PostmanRuntime/7.26.5` (from the [library](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent#library_and_net_tool_ua_strings) section of that page). Specifically, `User-Agent: <product>/<product-version>`.   I'm not sure if whitespace in the product name is conventional, so you might use `PrismMockServer/<version>`.

I think the rest in parenthesis is fine as you have.

Reply:
Here's an example of the new one as just committed:
`User-Agent: PrismMockServer/4.10.5 (Linux x64 5.10.102.1-microsoft-standard-WSL2)`